src/Clock/Clock.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  name: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const today = new Date();
+
+      this.setState({ today });
+      // eslint-disable-next-line no-console
+      console.info(today.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
```

Comment:
Instead of comparing entire `prevProps` and `this.props`, you should compare individual prop values (in this case, `prevProps.name !== this.props.name`) to avoid unnecessary console logs when other props change in the future. 🧐